src/api/controllers/FeedbackController.ts

```diff
@@ -0,0 +1,71 @@
+import { Body, Delete, Get, JsonController, Params, Post } from 'routing-controllers';
+
+import { FeedbackService } from '../../services/FeedbackService';
+import { CreateFeedbackRequest, ErrorResponse, getErrorMessage, GetFeedbackResponse, GetFeedbacksResponse, GetSearchedFeedbackRequest } from '../../types';
+import { UuidParam } from '../validators/GenericRequests';
+
+@JsonController('feedback/')
+export class FeedbackController {
+  private feedbackService: FeedbackService;
+
+  constructor(feedbackService: FeedbackService) {
+    this.feedbackService = feedbackService;
+  }
+
+  @Get()
+  async getAllFeedback(): Promise<GetFeedbacksResponse | ErrorResponse> {
+    try {
+      const feedback = await this.feedbackService.getAllFeedback();
+      return { feedback: feedback.map((f) => f.getFeedbackInfo()) };
+    } catch (error) {
+      return { error: getErrorMessage(error) }
+    }
+  }
+
+  @Get('id/:id/')
```

Comment:
how does this work? is the endpoint `/feedback/id/:id`?  This should be better named to conform to CRUD

Reply:
I'm following the same convention as for the other two models where the endpoint to get posts and users by id is `/post/id/:id/` and `/user/id/:id/`. How could I make it better?